example/mail.php

```diff
@@ -26,57 +30,93 @@
 $mail->Port = 587;   																	// TCP port to connect to
 
 $mail->setFrom($email);
-if ($sendCopy) $mail->addAddress($email);             // Copy sender if requested
-$mail->addAddress('directory@ontariotechu.ca');
 $mail->isHTML(true);                                  // Set email format to HTML
 
-$mail->Subject = 'Update Information in the Ontario Tech Directory.';
-$mail->Body  = "<p>Hello,
-<br/>I would like to update my contact information in the <a href=\"https://ontariotechu.ca/directory\">Ontario Directory</a>.</p>
-<p>Please use the following information to update my entry:</p>
-<table>
-<tbody>
-<tr>
-<td><strong>Banner ID:</strong></td>
-<td>$bannerId</td>
-</tr>
-<tr>
-<td><strong>First Name:</strong></td>
-<td>$firstName</td>
-</tr>
-<tr>
-<td><strong>Last Name:</strong></td>
-<td>$lastName</td>
-</tr>
-<tr>
-<td><strong>Department:</strong></td>
-<td>$department</td>
-</tr>
-<tr>
-<td><strong>Title:</strong></td>
-<td>$title</td>
-</tr>
-<tr>
-<td><strong>Building:</strong></td>
-<td>$building</td>
-</tr>
-<tr>
-<td><strong>Office:</strong></td>
-<td>$office</td>
-</tr>
-<tr>
-<td><strong>Extension:</strong></td>
-<td>$extension</td>
-</tr>
-<tr>
-<td><strong>E-mail:</strong></td>
-<td>$email</td>
-</tr>
-</tbody>
-</table>
-<p>Thank you for your assistance,<br/>
-<strong>$firstName $lastName</strong></p>
-";
+// check if the form is sent for message purpose or contact update purpose
```

Comment:
Could we condense this chunk a bit to remove some of the duplicated email template code? For instance, you could put the `<table>`'s contents that repeat in a separate variable, conditionally add the extra bits based on `$recipient`, and then place the contents inside the table after the fact.

```php
$tbody = "<tr>
  <td><strong>First Name:</strong></td>
  <td>$firstName</td>
  </tr>
  <tr>
  <td><strong>Last Name:</strong></td>
  <td>$lastName</td>
  </tr>
  <tr>
  <!-- etc etc -->";

$message = $recipient ? "" : "<p>Hello,
  <br/>I would like to update my contact information in the...";

$email = "$message<table><tbody>$tbody</tbody></table>";
```

Reply:
Good point!